### PR TITLE
PROVISION: Initialize Azure Resources for Blazor Project

### LINKED ISSUE #62: PROVISION: Initialize Azure Resources for Blazor Project
Here's some videos to help you understand the task:

https://www.youtube.com/watch?v=ht0xhQyF1x4

https://www.youtube.com/watch?v=E67QyXV_-v8&list=PLan3SCnsISTQ63o4HFkIw0YcEcOlwqt53&index=6
https://www.youtube.com/watch?v=96tl4f1gfa4&list=PLan3SCnsISTQ63o4HFkIw0YcEcOlwqt53&index=7
https://www.youtube.com/watch?v=sOje4-6psEY&list=PLan3SCnsISTQ63o4HFkIw0YcEcOlwqt53&index=8
https://www.youtube.com/watch?v=mryQEZIQqmA&list=PLan3SCnsISTQ63o4HFkIw0YcEcOlwqt53&index=9

### PR DESCRIPTION
**Pull request by rcruzfreelance**
Closes #62

### CODE REVIEW

File: OtripleS.Portal.Web.Infrastructure.Provision/Brokers/Clouds/CloudBroker.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                    clientSecret: this.clientSecret,
+                    tenantId: this.tenantId,
+                    environment: AzureEnvironment.AzureGlobalCloud);
+
+            return Azure.Configure()
+                .WithLogLevel(HttpLoggingDelegatingHandler.Level.Basic)
+                .Authenticate(credentials)
+                .WithDefaultSubscription();
+        }
+
```
Comment (hassanhabib) [inline OtripleS.Portal.Web.Infrastructure.Provision/Brokers/Clouds/CloudBroker.cs:44]: Bad

**Comment by hassanhabib** [comment]
@rcruzfreelance  please create a broker partial class for provisioning AppInsights

**Comment by rcruzfreelance** [comment]
@hassanhabib I might need help with this I tried to look in to the documentation [azure app insights](https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Microsoft.Azure.Management.ApplicationInsights/0.2.0-preview/api/Microsoft.Azure.Management.ApplicationInsights.Management.Models/Microsoft.Azure.Management.ApplicationInsights.Management.Models.ApplicationInsightsComponent.html) but theres no define methods for this like the other we have

**Comment by hassanhabib** [comment]
> @hassanhabib I might need help with this I tried to look in to the documentation [azure app insights](https://azuresdkdocs.blob.core.windows.net/$web/dotnet/Microsoft.Azure.Management.ApplicationInsights/0.2.0-preview/api/Microsoft.Azure.Management.ApplicationInsights.Management.Models/Microsoft.Azure.Management.ApplicationInsights.Management.Models.ApplicationInsightsComponent.html) but theres no define methods for this like the other we have

@rcruzfreelance sounds good I will setup some content/time for it

**Comment by hassanhabib** [comment]
I will be merging this as first step and add AppInsights to this effort in a different PR/Effort.

**Pull request merged**